packages/api-plugin-promotions/src/preStartup.js

```diff
@@ -0,0 +1,101 @@
+import SimpleSchema from "simpl-schema";
+import { Action, Trigger } from "./simpleSchemas.js";
+import noop from "./actions/noop.js";
+
+/**
+ * @summary apply all schema extensions to the Promotions schema
+ * @param {Object} context - The application context
+ * @returns {undefined} undefined
+ */
+function extendSchemas(context) {
+  const {
+    promotions: { schemaExtensions },
+    simpleSchemas: { Promotions }
+  } = context;
```

Comment:
Style nitpick. We do this on one line in like 1000 places in the app so would like to keep it consistent.